tests/test_examples.py

```diff
@@ -24,6 +24,13 @@
 
 import pytest
 
+try:
+    import __pypy__  # noqa
+except ImportError:
+    PYPY = False
+else:
+    PYPY = True
```

Comment:
Yeah, this is on master. I'll merge from master on the vcr branch and rebase this on that.